detect_secrets/audit/report.py

```diff
@@ -28,7 +28,7 @@ def generate_report(
     baseline_file: str,
     class_to_print: SecretClassToPrint = None,
     line_getter_factory: Callable[[str], 'LineGetter'] = open_file,
-) -> List[Dict[str, Any]]:
+) -> Dict[str, List[Any]]:
```

Comment:
I wonder if we can change it to `Dict[str, List[Dict[str, Any]]]` to maintain the typing benefits

Reply:
Yea, nice!